src/main/java/co/rsk/federate/signing/hsm/HSMVersion.java

```diff
@@ -0,0 +1,20 @@
+package co.rsk.federate.signing.hsm;
+
+public enum HSMVersion {
+    V1(1),
+    V2(2),
+    V3(3),
```

Comment:
We can do this in another task, but I propose we remove V3. There is no HSM v3 ever released

Reply:
There are lines of codes referencing hsm version 3, check [this](https://github.com/rsksmart/powpeg-node/blob/master/src/main/java/co/rsk/federate/signing/hsm/advanceblockchain/ConfirmedBlocksProvider.java#L97) and [this](https://github.com/rsksmart/powpeg-node/blob/master/src/main/java/co/rsk/federate/signing/hsm/advanceblockchain/HsmBookkeepingClientImpl.java#L169).

Should these conditionals be changed then?

Reply:
Yes, but in another task

Reply:
Task created.